c/planarityTestGraphFunctionality.c

```diff
@@ -408,15 +315,120 @@ int testAllGraphs(graphP theGraph, char command, char *inputStr, testAllStatsP s
 	if (freeG6ReadIterator(&pG6ReadIterator) != OK)
 		ErrorMessage("Unable to free G6ReadIterator.\n");
 	
-	return exitCode;
+	return Result;
 }
 
-int _getNumCharsToReprInt(int theNum)
+int outputTestAllGraphsResults(char command, testAllStatsP stats, char * infileName, char *outfileName, char **outputStr)
 {
-	int numCharsRequired = 1;
+	int Result = OK;
 
-	while(theNum /= 10)
-		numCharsRequired++;
+	FILE *outputFileP;
```

Comment:
This caught my eye as an uninitialized pointer. Normally, one would assign to NULL, but it is better for you to just declare it later, as mentioned below.  So, just delete this line.